apps/backoffice/mocks/handlers/services-cms-handlers.ts

```diff
@@ -64,126 +65,126 @@ export const buildHandlers = () => {
+        [ctx.status(409), ctx.json(getIoServicesError(400))],
+        [ctx.status(429), ctx.json(getIoServicesError(400))],
+        [ctx.status(500), ctx.json(getIoServicesError(400))]
       ];
 
       return res(...resultArray[0]);
     }),
     rest.patch(`${baseURL}/services/:serviceId/review`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(204)],
-        [ctx.status(400), ctx.json(null)],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(409), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(400), ctx.json(getIoServicesError(400))],
+        [ctx.status(401), ctx.json(getIoServicesError(400))],
+        [ctx.status(403), ctx.json(getIoServicesError(400))],
+        [ctx.status(404), ctx.json(getIoServicesError(400))],
+        [ctx.status(409), ctx.json(getIoServicesError(400))],
+        [ctx.status(429), ctx.json(getIoServicesError(400))],
+        [ctx.status(500), ctx.json(getIoServicesError(400))]
       ];
 
       return res(...resultArray[0]);
     }),
     rest.post(`${baseURL}/services/:serviceId/release`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(204)],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(409), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(401), ctx.json(getIoServicesError(400))],
+        [ctx.status(403), ctx.json(getIoServicesError(400))],
+        [ctx.status(404), ctx.json(getIoServicesError(400))],
+        [ctx.status(409), ctx.json(getIoServicesError(400))],
+        [ctx.status(429), ctx.json(getIoServicesError(400))],
+        [ctx.status(500), ctx.json(getIoServicesError(400))]
       ];
 
       return res(...resultArray[0]);
     }),
     rest.get(`${baseURL}/services/:serviceId/release`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(200), ctx.json(getGetPublishedService200Response())],
-        [ctx.status(401), ctx.json(null)],
-        [ctx.status(403), ctx.json(null)],
-        [ctx.status(404), ctx.json(null)],
-        [ctx.status(429), ctx.json(null)],
-        [ctx.status(500), ctx.json(null)]
+        [ctx.status(401), ctx.json(getIoServicesError(400))],
+        [ctx.status(403), ctx.json(getIoServicesError(400))],
+        [ctx.status(404), ctx.json(getIoServicesError(404))],
+        [ctx.status(429), ctx.json(getIoServicesError(400))],
+        [ctx.status(500), ctx.json(getIoServicesError(400))]
       ];
 
-      return res(...resultArray[0]);
+      return res(...resultArray[3]);
     }),
     rest.delete(`${baseURL}/services/:serviceId/release`, (_, res, ctx) => {
       const resultArray = [
         [ctx.status(204)],
```

Comment:
instead of having to map each return value from the backend, can't we just proxy the http call wihout doing this mapping?